README.md

```diff
@@ -25,6 +25,19 @@ pip install -e .
 picosvg mysvg.svg
 ```
 
+## Test
+
+```shell
+pip install pytest
+pytest
+```
+
+You can also use [pytest](https://docs.pytest.org/) to test the specified files individually.
+
+```shell
+pytest tests/svg_test.py
+```
+
```

Comment:
Best to reference https://googlefonts.github.io/python similar to Releasing. You might need to add this content there. Don't `pip install pytest`; prefer to `pip install -e .[dev]` before running tests

Reply:
Thanks for your review, I updated it.

Reply:
Done, PTAL